src/EventStore.Client/GrpcGossipClient.cs

```diff
@@ -6,11 +6,17 @@
 
 namespace EventStore.Client {
 	internal class GrpcGossipClient : IGossipClient {
-		public async ValueTask<ClusterMessages.ClusterInfo> GetAsync(ChannelBase channel,
-			CancellationToken cancellationToken) {
-			var client = new Gossip.Gossip.GossipClient(channel);
+		private readonly EventStoreClientSettings _settings;
+
+		public GrpcGossipClient(EventStoreClientSettings settings) {
+			_settings = settings;
+		}
 
-			using var call = client.ReadAsync(new Empty(), cancellationToken: cancellationToken);
+		public async ValueTask<ClusterMessages.ClusterInfo> GetAsync(ChannelBase channel, CancellationToken ct) {
+			var client = new Gossip.Gossip.GossipClient(channel);
+			using var call = client.ReadAsync(
+				new Empty(),
+				EventStoreCallOptions.CreateNonStreaming(_settings, ct));
```

Comment:
does this also affect the deadline used?

Reply:
yeah i think this is causing the deadline from settings to be used where previously one wasn't specified, so this is for the best